### DISCUSSION TITLE
Is all of this really necessary?

### DISCUSSION THREAD

**Discussion by jridder** [Homebrew/discussions#4763] [Getting started]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

It's really hostile. Those of us on beta's know that we need to expect problems. It was good enough to just say you are on a beta.

Warning: You are using macOS 14.
We do not provide support for this pre-release version.
It is expected behaviour that some formulae will fail to build in this pre-release version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.

**Comment by iMichka** [comment]
Hi. When we added this message, it massively reduced the requests we get about newer macOS versions.
People were almost harassing us on twitter, private email. I even sometimes get private messages on Linkedin...
Tens of issues were opened asking for help, which led to maintainers having to do more issue triaging than usual.

It's a trade off we made: the tone is a little bit rough, but it has helped maintainers focusing on the important things: making everything ready for the official release day. There is a lot of activity going on behind the scenes that our users might not see (like setting up the build machines, pre-bottling more than 6500 packages, ...). We need time for this and do not need a flood of tickets on top of that to manage :(

**Comment by SMillerDev** [comment]
> It's really hostile. Those of us on beta's know that we need to expect problems.

Looking for past posts here and in the issue trackers. I'm not so sure about that.

**Comment by ZhongRuoyu** [comment]
See also https://github.com/orgs/Homebrew/discussions/4739, if what you want is for the message to disappear.
